pkg/sources/postgres/dogs_test.go

```diff
@@ -113,7 +113,8 @@ func (s StoreTestSuite) TestUpdateDog() {
 		s.NoError(err)
 		createdDog.Name = "Lolita"
 
-		actualDog, err := s.store.UpdateDog(ctx, &dog)
+		s.NotEqual(dog.Name, createdDog.Name)
+		actualDog, err := s.store.UpdateDog(ctx, createdDog)
```

Comment:
This line was previously only sending in `dog`, rather than `createdDog`. So, that meant that both `actualDog` and `createdDog` were just new pointers to the same address as `dog`, and neither of them were validations of what was actually saved in the database.